lib/data/multipleBackendGateway.js

```diff
@@ -77,9 +147,36 @@ const multipleBackendGateway = {
                     }
                     cb();
                 });
+            } else if (client.clientType === 'aws_s3') {
+                client.headBucket({ Bucket: client.awsBucketName },
+                err => {
+                    if (err) {
+                        multBackendResp[location] = { error: err };
```

Comment:
We return this: ```'aws-test':
   { error:
      { NotFound: null
          at Request.extractError (/Users/nicolas.humbert/Desktop/s3/node_modules/aws-sdk/lib/services/s3.js:519:35)
          at Request.callListeners (/Users/nicolas.humbert/Desktop/s3/node_modules/aws-sdk/lib/sequential_executor.js:105:20)
          at Request.emit (/Users/nicolas.humbert/Desktop/s3/node_modules/aws-sdk/lib/sequential_executor.js:77:10)
         ...
message: null,
        code: 'NotFound',
        region: null,
        time: 2017-04-11T22:49:02.165Z,
        requestId: '0401E2616E70626B',
        extendedRequestId: 'PWNV0HrkokrIWTTzxkKXARf0z/r8YnrMA/DgCvUnKiimBSOsYpmzwmhLHH3sCYSW9YRWGzDLTzE=',
        cfId: undefined,
        statusCode: 404,
        retryable: false,
        retryDelay: 11.879374018464993 } } }```

It may be enough but adding `AWS S3 bucket is missing` or something similar before it may make it clearer.